content/sensu-enterprise/3.1/integrations/ec2.md

```diff
@@ -110,6 +113,84 @@ type         | Integer
 default      | `10`
 example      | {{< highlight shell >}}"timeout": 30{{< /highlight >}}
 
+## Cross-Account Access
+Cross-account access lets you use IAM-defined trust relationships to access a Sensu Enterprise instance from EC2 clients across multiple AWS accounts.
+
+### Client Configuration
+The EC2 integration supports account access configuration at the client level.
+To configure account access, add the `account` attribute to the Sensu client configuration within the `ec2` scope.
+
+#### Client Configuration Example
+
+{{< highlight json >}}{
+    "client": {
+      "name": "i-424242",
+      "subscriptions": ["production"],
+      "ec2": {
+          "account": "sensuapp"
+      }
+    }
+}{{< /highlight >}}
+
+#### `ec2` attributes
+
+account      | 
+-------------|------
+description  | The name of the account Sensu Enterprise should use to access EC2, used internally to Sensu only
+required     | false
+type         | String
+example      | {{< highlight shell >}}"account": "sensuapp"{{< /highlight >}}
+
+### Integration Configuration
+To enable cross-account support in the EC2 integration, add the `accounts` attributes, `name` and `role_arn`, to the EC2 integration configuration in Sensu.
+When processing events from clients with an `ec2.account` attribute, Sensu Enterprise applies the matching Amazon resource name (`role_arn`) stored in the integration configuration to access EC2.
+
+#### Integration Configuration Example
+
+{{< highlight json >}}
+{
+  "ec2": {
+    "region": "us-west-2",
+    "access_key_id": "AlygD0X6Z4Xr2m3gl70J",
+    "secret_access_key": "y9Jt5OqNOqdy5NCFjhcUsHMb6YqSbReLAJsy4d6obSZIWySv",
+    "allowed_instance_states": ["running"],
+    "timeout": 10,
+    "accounts": [
+      {
+        "name": "sensuapp",
+        "role_arn": "arn:aws:iam::921689585014:role/CrossAccountSignin"
```

Comment:
Can we obfuscate the ARN ? Like, just do "XXXXXXXXXXXXXX" instead of the number?